buildSrc/src/minimumRuntime/java/org/elasticsearch/gradle/VersionProperties.java

```diff
@@ -26,6 +26,7 @@ public static String getLucene() {
     public static String getBundledJdk(final String platform) {
         switch (platform) {
             case "darwin":
+            case "mac":
```

Comment:
Personally, I like to add a comment whenever I use fall-through so let future folks know this is purposeful and not a mistake.